variables.tf

```diff
@@ -310,3 +310,9 @@ variable "cognito_iam_role_arn" {
   default     = ""
   description = "ARN of the IAM role that has the AmazonESCognitoAccess policy attached"
 }
+
+variable "aws_ec2_service_name" {
+  type        = string
+  default     = ["ec2.amazonaws.com"]
```

Comment:
Here's an error. 
default value is list of strings where `type` is defined as a string.

Reply:
Thank you for the help. All done!